communication/marketing-team/README.md

```diff
@@ -1,57 +1,41 @@
-| sig-docs blog community liaison | TODO link to sig-docs handbook | onlydole |
-| Internal Comms Lead | TODO | |  
-
-
-How to get in contact with us:  
-TODO: add slack team handle  
-tag us on your issues and PRs in kubernetes/community with `area/contributor-comms`
-
-# Focus Areas  
+| Editor | TODO | @parispittman (to bootstrap); assoc editor/shadow: @mbbroberg |
+| Internal Comms Lead | TODO | @jonasrosland; shadow: @simplytunde |
+| Social Media Lead | TODO | @chris-short; shadow - [Could be you!] (paris to help bootstrap) |
+| Bloggers | TODO | @onlydole (liaison to sig-docs-blog team) @celanthe, @vonguard, @boluisa, [Could be you!] |
+| Designer | TODO | [Could be you!] help us with thumbnail graphics, digital assets, infographics, contributor site |
+
+## Contact Us  
+Reach out to an individual above via slack  
+Whole group - @contributor-comms in #sig-contribex on slack    
+Tag us in an issue in `kubernetes/community` with the label `area/contributor-comms`
+
+## Focus Areas  
 1.  inter-community group comms/project-wide contributor communications,   
 2.  contributor twitter/social and web (contributor site),   
 3.  blogging (work happening in SIGs, stories about contributors, recaps),  
 4.  program and event promotion (community meeting, meet our contributors, steering
   election, et al)  
 
-# Resources for the team
-[Entire community repo]  
-The future home of the [contributor site], [project board]  
-[surveys]  
-[Communication]  
-[style guide]  
-[Org structure] and [Community Groups List]
-
-[Values] and [principles] of the project  
-[YouTube playlist] TODO add  
-`area/contributor-comms` - label for issues and PRs  
-
-
-[contributor site]: https://github.com/kubernetes-sigs/contributor-site
-[project board]: https://github.com/orgs/kubernetes-sigs/projects/3
-[surveys]: https://github.com/kubernetes/community/tree/master/sig-contributor-experience
-[Communication]: https://github.com/kubernetes/community/tree/master/communication
-[style guide]: https://git.k8s.io/community/contributors/guide/style-guide.md
-[Org structure]: https://github.com/kubernetes/community/blob/master/governance.md
-[Community Groups List]: https://github.com/kubernetes/community/blob/master/sig-list.md
-[Values]: https://github.com/kubernetes/community/blob/master/values.md
-[principles]: https://github.com/kubernetes/community/blob/master/governance.md#principles
-[Entire community repo]: https://github.com/kubernetes/community
-[marketing folder]:  https://github.com/kubernetes/community/blob/master/communication/marketing-team/CHARTER.md
+## Could be you!
+We still have many roles available!  
+
+Come to one of our [meetings] (join kubernetes-sig-contribex@googlegroups.com for the invite)
+or reach out to us on the #sig-contribex channel on the Kubernetes slack (slack.k8s.io for invite)
+Let us know you are interested and if you have any questions.
+
+
+[meetings]: https://github.com/kubernetes/community/blob/master/sigs.yaml#L1143
```

Comment:
```suggestion
[meetings]: /sig-contributor-experience/README.md#community-management
```

This will point to the wrong thing after a sigs.yaml update 😬 Probably better to link to the community-management project in the readme.

Reply:
fixed